### PR TITLE
[#143] Add ignored path globs as a config

### LINKED ISSUE #143: Add ignored areas to the glob find to improve performance
## Issue:
- currently the file loader is very slow when being used outside npm projects
- take the top languages for the extension and use their package management folders as a "sane" default.
  - `node_modules, venv, vendors`
- use these sane defaults in an extension setting to be used in the glob find https://github.com/ryanluker/vscode-coverage-gutters/blob/master/src/filesloader.ts#L83-L98

### PR DESCRIPTION
**Pull request by Tadaboody**
Starting from this commit, ignored paths are able to be edited using the
extension settings

Not sure how to test this though

Closes #143

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -124,6 +124,15 @@
...
           "description": "show or hide the gutter coverage"
         },
+        "coverage-gutters.ignoredPathGlobs": {
+          "type": "Array<string>",
+          "default": [
+            "**/node_modules/**",
+            "**/venv/**",
+            "**/vendor/**"
+          ],
+          "description": "show or hide the gutter coverage"
```
Comment (ryanluker) [inline package.json]: probably give this a better name related to the ignoring of the paths.

File: test/fakeConfig.test.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { IConfigStore } from "../src/extension/config";
+
+export const fakeConfig: IConfigStore = {
```
Comment (ryanluker) [inline test/fakeConfig.test.ts]: nice cleanup here, great work!

**Comment by ryanluker** [comment]
@Tadaboody I think just having the integration tests pass for this is good enough as we dont really want to test the glob module. Thanks again for submitting a pull request!

File: test/fakeConfig.test.ts (hunk 2 above)
Comment (Tadaboody) [reply test/fakeConfig.test.ts]: Thanks!

**Comment by Tadaboody** [comment]
@ryanluker changes were introduced in https://github.com/ryanluker/vscode-coverage-gutters/pull/168/commits/887a9fd2e26cc263f7be8652b9d828cf04999d0f. Awaiting re-review.

**Comment by ryanluker** [comment]
@Tadaboody thanks for helping with the extension!

**Pull request merged**